Resources/Prototypes/DeltaV/Roles/Jobs/Justice/chief_justice.yml

```diff
@@ -15,7 +15,7 @@
       min: 36000 # 10 hours
     - !type:CharacterOverallTimeRequirement
       min: 90000 # 25 hours
-    - !type:WhitelistRequirement # whitelist requirement because I don't want any dingus judges
+    - !type:CharacterWhitelistRequirement # whitelist requirement because I don't want any dingus judges
```

Comment:
um

Reply:
Oh, I had to fix those on my own branch in [4f8f84d](https://github.com/Simple-Station/Einstein-Engines/pull/733/commits/4f8f84dab60feeb7a686a5490e98a33f279e3cd0) because the game wouldn't load due to requirements being invalid. I expected all of the changes to cancel out later, but looks like this one didn't get fixed in base EE.

Reply:
honestly i'm fine enough with this fixing that